docs/topic/billing/chargeable-resources.md

```diff
@@ -0,0 +1,639 @@
+
+Count thus becomes the most common determiner of how much the CPU component
+of a node costs for our usage pattern.
+
+### Memory
+
+Unlike CPU, memory is simpler - capacity is what matters. A GB of memory is a GB
+of memory! We pay for how much memory capacity the VM we request has, regardless
+of whether we use it or not.
+
+### Boot disk (ephemeral)
+
+Each node needs some disk space that lives only as long as the node is up,
+to store:
+
+1. The basic underlying operating system that runs the node itself (Ubuntu,
+   [COS](https://cloud.google.com/container-optimized-os/docs) or similar)
+2. Storage for any docker images that need to be pulled and used on the node
+3. Temporary storage for any changes made to the *running containers* in a
+   pod. For example, if a user runs `pip install` or `conda install` in a
+   user server, that will install additional packages into the container
+   *temporarily*. This will use up space in the node's disk as long as the
+   container is running.
+4. Temporary storage for logs
+5. Any [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir)
+   volumes in use on the node. These volumes are deleted when the pod using
+   them dies, so are temporary. But while they are in use, there needs to
+   be 'some place' for this directory to live, and it is on the node's
+   ephemeral disk
+6. For *ephemeral hubs*, their home directories are also stored using the
+   same mechanism as (3) - so they take up space on the ephemeral disk as
+   well. Most hubs have a *persistent* home directory set up, detailed
+   in the [Home directories section](topic:billing:resources:home).
+
+
+These disks can be fairly small - 100GB at the largest, but we can get away
+with far smaller disks most of the time. The performance of these disks
+*primarily* matters during initial docker image pull time - faster the
+disk, faster it is to pull large docker images. But it can be really
+expensive to make the base disks super fast SSDs, so often a balanced
+middling performance tier is more appropriate.
+
+### Accelerators (GPU, etc)
+
+Accelerators are a form of [co-processor](https://en.wikipedia.org/wiki/Coprocessor) -
+dedicated hardware that is more limited in functionality than a general
+purpose CPU, but much, *much* faster. [GPU](https://en.wikipedia.org/wiki/Graphics_processing_unit)
+are the most common ones in use today, but there are also [TPU](https://cloud.google.com/tpu?hl=en)s,
+[FGPA](https://en.wikipedia.org/wiki/Field-programmable_gate_array)s
+available in the cloud. Accelerators can often only be attached to a subset of a cloud provider's available node types, node sizes, and cloud zones.
+
+GPUs are the most commonly used with JupyterHubs, and are often the most
+expensive as well! Leaving a GPU running for days accidentally is the
+second easiest way to get a huge AWS bill (you'll meet NAT / network egress, the primary culprit, later in this document). So they are usually segregated
```

Comment:
> a huge AWS bill

Or any other cloud provider's bill as long as they offer GPUs (super minor, no need to fix, IMHO).